docs/source/markdown/podman-system-service.1.md

```diff
@@ -13,6 +13,10 @@ If no endpoint is provided, defaults will be used.  The default endpoint for a r
 service is *unix:/run/podman/podman.sock* and rootless is *unix:/$XDG_RUNTIME_DIR/podman/podman.sock* (for
 example *unix:/run/user/1000/podman/podman.sock*)
 
+To access the API service inside a container, you need to
+- mount the socket as a volume
+- run the container with `--privileged`
```

Comment:
This is is not necessary.

--security-opt label:disable ...

Should be enough.

Reply:
Done